hosts.txt

```diff
@@ -12291,6 +12291,10 @@
 127.0.0.1 beacon.tru.am
 
 # [truecaller.com]
+127.0.0.1 ads5.truecaller.com
+127.0.0.1 ads5-noneu.truecaller.com
+127.0.0.1 ads-router-noneu.truecaller.com
+127.0.0.0 batchlogging4.truecaller.com
```

Comment:
@spirillen plz check this #220

Reply:
Whoopsie :smile:

Reply:
Should be fixed in https://github.com/AdAway/adaway.github.io/pull/221